soc/infineon/cat1a/soc.yml

```diff
@@ -129,3 +114,12 @@ family:
     - name: cy8c6144lqi_s4f62
     - name: cy8c6144azi_s4f12
     - name: cy8c6144lqi_s4f12
+    - name: cyble_416045_02_device
```

Comment:
why does this now have device in the name?

Reply:
I had a quick search for this and can't find any such soc on your website?

Reply:
cyble_416045_02 this is module based on PSoC 63 https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-bluetooth-le-bluetooth-multiprotocol/airoc-bluetooth-modules/cyble-416045-02/

In PDL, modules has "_device" ending in device headers:   https://github.com/Infineon/mtb-pdl-cat1/blob/master/devices/COMPONENT_CAT1A/include/cyble_416045_02_device.h

To user prevent confusing i changed :  cyble_416045_02_device -> cyble_416045_02

in [modules/hal_infineon/mtb-pdl-cat1/CMakeLists.txt](https://github.com/zephyrproject-rtos/zephyr/compare/ffacb3614304cf6411d00a25fef9d1d32ddeca8d..324a0fb218b441c4ad8778b4325515d286927ef4#diff-c4402b261bf1d521423ef6072e0640a118e7c655828defc67eae86f0a99c6a78) i added  zephyr_compile_definitions_ifdef for CONFIG_SOC_CYBLE_416045_02  to handling PDL related devines..
```
# Generate PDL specific define (w. *_device) for SOC module (e.g: CYBLE_416045_02) 
zephyr_compile_definitions_ifdef(CONFIG_SOC_CYBLE_416045_02 CYBLE_416045_02_device)
```